foundations/06-cascade-fix/solution/solution.html

```diff
@@ -9,15 +9,15 @@
     <link rel="stylesheet" href="solution.css">
   </head>
   <body>
-    <p class="para">I'm just a paragraph with red text!</p>
-    <p class="para small-para">I'm a smaller paragraph, also with red text!</p>
+    <p class="para">I'm just a paragraph with bold text!</p>
+    <p class="para small-para">I'm a smaller paragraph, also with bold text!</p>
```

Comment:
```suggestion
    <p class="para small-para">I'm a smaller paragraph, also with extra bold text!</p>
```
**suggestion:** Adding the "extra" to the wording in the `solution.html`. You can commit these right from the comments if you'd like!